docs/content/configuration.md

```diff
@@ -1,12 +1,11 @@
 # Configuration
 
-The configuration for maesh is broken down into two parts: the static configuration, and the dynamic configuration.
-The static configuration is configured when the maesh service mesh is installed,
-and is configured via the values.yaml file in the helm install.
+The configuration for Maesh is broken down into two parts: the static configuration, and the dynamic configuration.
+The static configuration is configured when the Maesh service mesh is installed, and is configured via the values.yaml file in the helm install.
 
 ## Static configuration
 
-- The maesh image version can be manually defined if needed, as can the version for the Traefik CE mesh nodes.
+- The Maesh image version can be manually defined if needed, as can the version for the Traefik CE mesh nodes.
```

Comment:
Can we rename Traefik CE to just Traefik, please? :)

Reply:
done